### PR TITLE
Hello World in Ferret

### LINKED ISSUE #502: Add Hello World in Ferret
See: https://github.com/nakkaya/ferret

### PR DESCRIPTION
**Pull request by ArfatSalman**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [x] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [x] I named the pull request using `<Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  - [x] I added fun facts (i.e. debut, developer, typing, etc.)
  - [x] I added reference link(s) to the README
  - [x] I added solution citations when necessary (see [plagiarism][2])
- [x] I fixed #502 

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

File: archive/f/ferret/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Sample Programs in Ferret
+
+Welcome to Sample Programs in Ferret!
+
+Ferret takes in Clojure files and compiles them to C++.
+
+## Sample Programs
+
+- [Hello World in Ferret](https://github.com/TheRenegadeCoder/sample-programs/issues/502)
```
Comment (jrg94) [inline archive/f/ferret/README.md]: Can you create a new issue for this line? We usually create a separate issue for the missing article and link it here.

File: archive/f/ferret/hello-world.clj [hunk 2]
```diff
@@ -0,0 +1,3 @@
+(do
+  (println "Hello")
```
Comment (jrg94) [inline archive/f/ferret/hello-world.clj]: Any particular reason for the println? For consistency, we stick with "Hello, World!".

Comment (ArfatSalman) [reply archive/f/ferret/hello-world.clj]: I saw Hello world in Java and it used a `println`. Hence, I used that too.

I'll update the text to Hello, World.

File: archive/f/ferret/README.md (hunk 1 above)
Comment (ArfatSalman) [reply archive/f/ferret/README.md]: I'll do.

File: archive/f/ferret/hello-world.clj (hunk 2 above)
Comment (jrg94) [reply archive/f/ferret/hello-world.clj]: Sorry, I meant the string more than the usage. Println is fine!

**Comment by ArfatSalman** [comment]
Made the changes. Please review again. 

Thanks

**Pull request merged**